apier/v1/thresholds.go

```diff
@@ -79,6 +79,9 @@ func (apierV1 *ApierV1) SetThresholdProfile(thp *engine.ThresholdProfile, reply
 	if err := apierV1.DataManager.SetThresholdProfile(thp, true); err != nil {
 		return utils.APIErrorHandler(err)
 	}
+	if err := apierV1.DataManager.SetThreshold(&engine.Threshold{Tenant: thp.Tenant, ID: thp.ID}); err != nil {
```

Comment:
Please add a check, making sure we create new threshold if none existing already since Set can update an existing and we will destroy previus